### ISSUE TITLE
App does not load after JWT expires

### ISSUE THREAD

**Issue by paraterrestrial** [relata/relata-prototype#6]
After JSON Web Token expires, the app does not load and throws an uncaught 401 error with message `jwt expired`. After refreshing the page, the app loads successfully.

**Comment by paraterrestrial** [comment]
Whenever someone has an expired JWT token and opens the app, the app crashes. In dev mode, it shows the error `Unhandled Rejection (NonAuthenticated): jwt expired`.  If the user refreshes the page, the app loads fine with the user logged out.

The error seems to be thrown by the `client.reAuthenticate call` inside the login function in `frontend/src/components/App.js`.

But, no matter how I try to catch the error, I still get the unhandled
rejection.

I've also tried checking the validity of the JWT token via `client.getAccessToken()` before calling `reAuthenticate`.  But the problem is that once I call `getAccessToken`, it seems to remove the access token, and subsequent calls to `reAuthenticate fail`.  If I call `getAccessToken` twice, the second time it returns null.

(For debugging, I've changed the JWT expiry time from `1d` to `10s` in `config/default.json`, which works fine.)
